app/workers/hca/submission_failure_email_analytics_job.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+module HCA
+  class SubmissionFailureEmailAnalyticsJob
+    include Sidekiq::Worker
+
+    EMAIL_SUBJECT = "We didn't receive your application"
```

Comment:
seems fragile, is there no other way to do it?

Reply:
We do use that subject line in the Mailer too, so good point.  We should set it one place.

Reply:
maybe we can add a comment in the mailer that says if the subject is changed to remember to change this job too to include the old subject